packages/x-components/src/components/animations/create-directional-animation-factory.ts

```diff
@@ -1,4 +1,14 @@
-import Vue, { VueConstructor } from 'vue';
+import { defineComponent, h } from 'vue';
+
+type AnimationOrigin =
```

Comment:
I think we should keep exporting the type
```suggestion
export type AnimationOrigin =
```

Reply:
For any specific case? If not, I think it could be internal because when you use the returned function, the parameter will be type implicitly `animateTranslate('bottom');`. If we don't need to use outside explicitly, we could keep it internal

Reply:
I had issues building the project locally without exporting the type. However might be fine but then we need to label it as BREAKING-CHANGE

Reply:
You are right! It will be a BC. Consider it exported